lib/controllers/frontend/spree/checkout_controller_decorator.rb

```diff
@@ -1,4 +1,3 @@
-require 'spree/core/validators/email'
```

Comment:
@artplan1 we still need this for Spree < 3.5, so we need a check here eg. 

```
require 'spree/core/validators/email' unless Spree.version.to_f > 3.5
```

Reply:
EmailValidator was removed from that controller in c199b6c#diff-6d07fe992144746adf8f712814389d5c

I checked it in all branches of this `spree_auth_devise` gem, it's not in use in that controller